web-services-app/src/main/java/edu/unc/lib/boxc/web/services/processing/IiifV3ManifestService.java

```diff
@@ -45,6 +45,7 @@
 import java.util.Objects;
```

Comment:
we also need to update `setSoundContent` so that when there is a audio access copy, it will set the download path to the audio access copy instead of the original file. This should tell IIIF viewers to display the access copy instead of the original. We should make sure to verify that in a test as well.